docs/spec/api.md

```diff
@@ -16,7 +16,7 @@ of this API, known as _Docker Registry HTTP API V2_.
 
 While the V1 registry protocol is usable, there are several problems with the
 architecture that have led to this new version. The main driver of this
-specification is a set of changes to the docker the image format, covered in
+specification is a set of changes to the Docker image format, covered in
```

Comment:
can you also make the same change to the `.tmpl` file in the same directory? https://github.com/docker/distribution/blob/master/docs/spec/api.md.tmpl#L19 (IIIRC this file is generated from that one)

Reply:
@thaJeztah Done. Original commit amended and pushed to the PR branch.